File: multiples-of-3-and-5/main.js
function main() {
  const n = 1000;
  let total = 0;

  for (let i = 0; i < n; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      total += i;
    }
  }
  console.log(total);
}

main();

File: multiples-of-3-and-5/main.scala
object Main extends App {
  val n: Int = 1000
  var total: Int = 0

  var i = 0
  for(i <- 1 to n ) {
    if(i % 3 == 0 || i % 5 == 0){
      total += i
    }
  }
  println(total);
}